transmart-rest-api/README.md

```diff
@@ -23,6 +23,7 @@ The return message's body will contain JSON with additional HAL format, with the
 | get observations for single concept | GET `/studies/{studyId}/concepts/{conceptPath}/observations` | returns clinical observation values for one concept for a study |
 | create patient set | POST `/patient_sets` | body should be query definition in a subset of i2b2's XML schema. Response is the same as for GET `/patient_sets/{result_instance_id}`  |
 | get single patient set | GET `/patient_sets/{resultInstanceId}` | returns information a previously created patient set |
+| disable patient set | POST `/patient_sets/{resultInstanceId}` | disables information a previously created patient set |
```

Comment:
From the rest-api client's point of view it gets deleted. I think the rest spec explicitly allows this kind of behavior of deletion archiving a resource in some way without physically removing it.